tests/util/models/gpflow/models.py

```diff
@@ -116,6 +116,10 @@ def get_kernel(self) -> tfp.math.psd_kernels.PositiveSemidefiniteKernel:
         return self.kernel
 
 
+class GPRcopy(GPR):
```

Comment:
Is this only used for testing the registry? If so it probably belongs there rather than here (as it's not a generally useful class).